blueprint/core/blueprint_EcmascriptEngine.cpp

```diff
@@ -282,63 +293,61 @@ namespace blueprint
                 {
                     const auto cb = f->second;
                     std::invoke(cb.f, juce::var::NativeFunctionArgs(juce::var(), cb.args.data(), static_cast<int>(cb.args.size())));
-                    if(!cb.repeats)
+                    if (! cb.repeats)
                     {
                         stopTimer(id);
                         timeoutFunctions.erase(f);
                     }
                 }
             }
 
-            private:
-                struct TimeoutFunction
-                {
-                    TimeoutFunction(const juce::var::NativeFunction _f, const std::vector<juce::var> &&_args, const bool _repeats=false)
+        private:
+            struct TimeoutFunction
+            {
+                TimeoutFunction(const juce::var::NativeFunction _f, const std::vector<juce::var>&& _args, const bool _repeats = false)
                     : f(_f), args(std::move(_args)), repeats(_repeats) {}
 
-                    const juce::var::NativeFunction f;
-                    std::vector<juce::var> args;
-                    const bool repeats;
-                };
+                const juce::var::NativeFunction f;
+                std::vector<juce::var> args;
+                const bool repeats;
+            };
 
-                std::map<int, TimeoutFunction> timeoutFunctions;
+            std::map<int, TimeoutFunction> timeoutFunctions;
         };
 
         // IsSetter is true for setTimeout / setInterval
         // and false for clearTimeout / clearInterval
         template <bool IsSetter = false, bool Repeats = false, typename MethodType>
         void registerNativeTimerFunction(const char* name, MethodType method)
         {
-            registerNativeProperty(name, juce::var::NativeFunction([this, name, method] (const juce::var::NativeFunctionArgs& _args) -> juce::var {
-                if constexpr (IsSetter)
-                {
-                    if(_args.numArguments < 2 || !_args.arguments[0].isMethod() || !_args.arguments[1].isDouble())
-                        throw Error(juce::String(name) + " requires a callback and time in milliseconds");
-                    // build a vector holding all additional arguments
-                    std::vector<juce::var> args(_args.arguments + 2, _args.arguments + _args.numArguments);
-                    return (this->timeoutsManager.get()->*method)(_args.arguments[0].getNativeFunction(), _args.arguments[1], std::move(args), Repeats);
-                }
-                else
-                {
-                    if(_args.numArguments < 1 || !_args.arguments[0].isDouble())
-                        throw Error(juce::String(name) + " requires an integer ID of the timer to clear");
-                    return (this->timeoutsManager.get()->*method)(_args.arguments[0]);
-                }
-            }));
+            registerNativeProperty(name, juce::var::NativeFunction([this, name, method](const juce::var::NativeFunctionArgs& _args) -> juce::var {
```

Comment:
I'm not a big fan of the way this indents the inline lambda here... is there a way to nudge this back as it was?